geopandas/tests/test_geoseries.py

```diff
@@ -297,6 +298,29 @@ def test_from_wkb_series_with_index(self):
         expected = self.g1.reindex(index)
         assert_geoseries_equal(expected, GeoSeries.from_wkb(s, index=index))
 
+    @pytest.mark.parametrize("input_type", ["str", "bytes"])
+    def test_from_ewkb(self, input_type):
+        srid = 4326
+        expected = self.g1.copy().set_crs(srid)
+        t1_wkb_elem = WKBElement(self.t1.wkb, srid=srid).as_ewkb()
+        sq_wkb_elem = WKBElement(self.sq.wkb, srid=srid).as_ewkb()
```

Comment:
Can we not depend on geoalchemy2 in basic tests and depend on it only optionally when we want to test it explicitly? We don't have geoalchemy2 in most of our CI envs and I'd like to keep that like that. You can use shapely  (set_srid) directly instead.